near/kycdao-ntnft/src/lib.rs

```diff
@@ -546,7 +717,7 @@ mod tests {
         let mut contract = KycdaoNTNFT::new_default_meta("base3".to_string());
 
         // use default status fallback
-        contract.authorize_minting(6547, accounts(4), sample_token_metadata("somehash".to_string()), Some(9000000000));
+        contract.authorize_mint_with_code(6547, accounts(4), sample_token_metadata("somehash".to_string()), Some(9000000000), 0, "KYC_1".to_string());
```

Comment:
I didn't see any tests that actually check payment, did I miss them?

Reply:
Also coming in a separate PR, sorry I couldn't finish them yet